rest-api-spec/src/main/resources/rest-api-spec/test/index/90_flat_object.yml

```diff
@@ -610,6 +610,51 @@ teardown:
       version: " - 2.99.99"
       reason: "flat_object is introduced in 3.0.0 in main branch"
 
+  # Mapping parameters (such as index/search analyzers) are currently not supported
+  # The plan is to support them in the next version
+  - do:
+      catch: bad_request
+      indices.create:
+        index: test_analyzer
+        body:
+          mappings:
+            properties:
+              data:
+                type : "flat_object"
+                analyzer: "standard"
+
+  - match: { error.root_cause.0.type: "mapper_parsing_exception" }
+  - match: { error.root_cause.0.reason: "Mapping definition for [data] has unsupported parameters:  [analyzer : standard]"}
+  - match: { status: 400 }
+
+  #  - do:
```

Comment:
@lukas-vlcek Hi Lukas, I am going to remove the commented out part here in the next commit

Reply:
Sure, you can. We can add it once we implement the mapping parameters.

Reply:
@lukas-vlcek Thanks Lukas adding the tests for analyzers, and match the exception messages. `unsupported parameters:  [analyzer : ` I did the clean up by the way in the commit [remove IndexAnalyzer from Builder](https://github.com/opensearch-project/OpenSearch/pull/6507/commits/543755fd1608c62685b2897fd41826148dd1b11d). Nice teamwork！

Reply:
> Sure, you can. We can add it once we implement the mapping parameters.

I only deleted the commented out part which is not finished. The testing analyzers parts are kept nicely.